src/test/java/uk/ac/ebi/eva/bd2k/transform/EvaStudyTransformerTest.java

```diff
@@ -59,18 +60,23 @@
 
     private VariantStudy variantStudy;
 
+    private String evaWebUrl;
+
+
     @Before
     public void setUp() throws Exception {
         projectClientMock = projectId -> new EnaProject(projectId, EVA_STUDY_PUBLICATION_DATE);
         preEvaProjectClientMock = projectId -> new EnaProject(projectId, PRE_EVA_STUDY_PUBLICATION_DATE);
 
         variantStudy = new VariantStudy("S1", "Study 1", "This is the study 1", "EBI", "Homo sapiens",
                                         new URI("http://www.study1.org"), "Illumina", "Case-Control");
+
+        evaWebUrl = "http://eva-host/eva/?eva-study=S1";
     }
 
     @Test
     public void transform() throws Exception {
-        EvaStudyTransformer studyTransformer = new EvaStudyTransformer(projectClientMock);
+        EvaStudyTransformer studyTransformer = new EvaStudyTransformer(projectClientMock, evaWebUrl);
```

Comment:
I think this constructor should receive the unformatted URL (http://eva-host/eva/?eva-study={0}) to mimic the tested code real behaviour